apps/dapp-console/app/components/Header/SignInButton.tsx

```diff
@@ -0,0 +1,60 @@
+'use client'
+
+import {
+  DropdownMenu,
```

Comment:
the https://github.com/ethereum-optimism/ecosystem/blob/main/packages/ui-components/src/components/ui/dropdown-menu.tsx file feels harder to read than the way our pragmatism components are laid out

Reply:
how do u usually use it